### PR TITLE
Modifying test_target_data_map.c to use headerfile (Issue #10)

### PR DESCRIPTION
**Pull request by tmh97**
Including macro usage, currently passing on clang and gcc, needs more work before review.

### CODE REVIEW

**Comment by josemonsalve2** [comment]
All, should we separate this test into multiple ones?

File: tests/4.5/target_data/test_target_data_map.c [hunk 1]
```diff
@@ -90,19 +85,15 @@ int test_map_tofrom() {
...
+  OMPVV_TEST_AND_SET_VERBOSE(errors, (N != sum) || (N != sum2));
 
   return errors;
 }
 
 // Test for OpenMP 4.5 target data map(to: ) 
 int test_map_to() {
 
-  printf("test_map_to\n");
+  OMPVV_INFOMSG("test_map_to\n");
```
Comment (josemonsalve2) [inline tests/4.5/target_data/test_target_data_map.c]: remove the `\n`

File: tests/4.5/target_data/test_target_data_map.c [hunk 2]
```diff
@@ -45,21 +45,17 @@ int test_map_from() {
...
+  OMPVV_TEST_AND_SET_VERBOSE(errors, (N != sum) || (2*N != sum2));
 
   return errors;
 }
 
 // Test for OpenMP 4.5 target data map(tofrom: ) 
 int test_map_tofrom() {
 
-  printf("test_map_tofrom\n");
+  OMPVV_INFOMSG("test_map_tofrom\n");
```
Comment (josemonsalve2) [inline tests/4.5/target_data/test_target_data_map.c]: remove `\n`

File: tests/4.5/target_data/test_target_data_map.c [hunk 3]
```diff
@@ -202,19 +187,16 @@ int test_map_to_from() {
...
+  OMPVV_TEST_AND_SET_VERBOSE(errors, ((N - sum) != 0));
 
   return errors;
 }
 
 // Test for OpenMP 4.5 target data map(alloc:)
 int test_map_alloc() {
 
-  puts("test_map_alloc");
+  OMPVV_INFOMSG("test_map_alloc\n");
```
Comment (josemonsalve2) [inline tests/4.5/target_data/test_target_data_map.c]: remove `\n`

File: tests/4.5/target_data/test_target_data_map.c [hunk 4]
```diff
@@ -163,21 +153,17 @@ int test_map_to() {
...
+  OMPVV_TEST_AND_SET_VERBOSE(errors, (N != sum) || (N != sum2));
 
   return errors;
 }
 
 // Test for OpenMP 4.5 target data map(to: ) and map(from:)
 int test_map_to_from() {
 
-  printf("test_map_to_from\n");
+  OMPVV_INFOMSG("test_map_to_from\n");
```
Comment (josemonsalve2) [inline tests/4.5/target_data/test_target_data_map.c]: remove `\n

File: tests/4.5/target_data/test_target_data_map.c [hunk 5]
```diff
@@ -229,7 +211,6 @@ int test_map_alloc() {
   {
 #pragma omp target is_device_ptr(d_sum, d_ishost)
```
Comment (josemonsalve2) [inline tests/4.5/target_data/test_target_data_map.c]: `d_ishost` not needed at all. Remove declaration as well

**Comment by tmh97** [comment]
> All, should we separate this test into multiple ones?

Let

**Comment by tmh97** [comment]
Fixed style issues and other problems in the last commit, currently passing on GCC and Clang. Let me know whether it is best to split these tests up!

**Comment by spophale** [comment]
Ideally I would have recommended splitting these tests.

**Comment by tmh97** [comment]
Tests separated, all individually pass on clang and gcc

**Pull request merged**